servers/zms/schema/updates/update-20201109.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS `zms_server`.`role_tags` (
+  `role_id` INT UNSIGNED NOT NULL,
+  `tag_key` VARCHAR(64) NOT NULL,
```

Comment:
since the table itself is already called role_tags, I don't think we need to specify tag_key/tag_value as column names, just having key/value is sufficient.